src/physics/physics_model.cpp

```diff
@@ -127,12 +159,30 @@ ::convective_numerical_split_flux(const std::array<real,nstate> &conservative_so
                                   const std::array<real,nstate> &conservative_soln2) const
 {
     // TO DO: Update for when nstate > nstate_baseline_physics
+    //std::array<dealii::Tensor<1,dim,real>,nstate> conv_num_split_flux;
+    //if constexpr(nstate==nstate_baseline_physics) {
+    //    conv_num_split_flux = physics_baseline->convective_numerical_split_flux(conservative_soln1,conservative_soln2);
+    //} else {
+    //    // TO DO, make use of the physics_model object for nstate>nstate_baseline_physics
+    //    std::abort();
+    //}
```

Comment:
Remove commented lines of code like this; applies throughout the file

Reply:
fixed